components/ContentRenderer/ContentRenderer.module.scss

```diff
@@ -84,6 +84,16 @@ $avatar-size: 60px;
             color: var(--prezly-accent-color);
             font-weight: $font-weight-bold;
         }
+
+        .prezly-slate-button-block {
+            background-color: var(--prezly-accent-color);
+            color: $color-base-white;
```

Comment:
What about `:hover` state?

I can see that the subscribe button does have it:

[Screencast from 2023-08-11 14-54-59.webm](https://github.com/prezly/theme-nextjs-bea/assets/370680/065feb7f-06d1-4120-bee8-5a251dc9c988)

Reply:
Updated